src/v3/src/components/Images/DeviceIcon.tsx

```diff
@@ -10,40 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { Theme } from '@mui/material';
 import { FunctionComponent, h } from 'preact';
 
 import { loc } from '../../util';
+import { DevicesIcon } from '../Icon/OdysseyIcons';
 
 export const DeviceIcon: FunctionComponent = () => (
-  <svg
-    width="16"
-    height="16"
-    viewBox="0 0 16 16"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-    role="img"
-    aria-hidden="true"
-  >
-    <title>
-      {loc('icon.title.browser', 'login')}
-    </title>
-    <path
-      d="M4.5 14.5H8M11.5 14.5H8M8 14.5V13"
-      stroke="#1662DD"
-      class="siwIconStrokePrimary"
-    />
-    <path
-      d="M3 1.5H13C13.8284 1.5 14.5 2.17157 14.5 3V10C14.5 10.8284 13.8284 11.5 13 11.5H3C2.17157 11.5 1.5 10.8284 1.5 10V3C1.5 2.17157 2.17157 1.5 3 1.5Z"
-      stroke="#1662DD"
-      class="siwIconStrokePrimary"
-    />
-    <rect
-      x="2"
-      y="9"
-      width="12"
-      height="1"
-      fill="#1662DD"
-      class="siwIconFillPrimary"
-    />
-  </svg>
+  <DevicesIcon
+    sx={(theme: Theme) => ({
+      color: theme.palette.primary.main,
+      width: '16px',
+      height: '16px',
```

Comment:
It's probably fine to leave this style for `width` and `height` right now since that's parity with the original custom SVG icon we were building, but I think we're going to want to reassess if we should be setting `width` and `height` or not because Odyssey does not explicitly set a `width` and `height` and instead uses MUI default values of `1em`

Reply:
I'll create a follow-up ticket in the epic to look at any icon sizing differences between icons that are unique to the SIW and icons that we get from Odyssey